mne_qt_browser/_pg_figure.py

```diff
@@ -3387,7 +3387,7 @@ def _overview_mode_changed(self, new_mode):
         if not self.mne.overview_bar.isVisible():
             self._toggle_overview_bar()
 
-    def _overview_radio_clicked(self, *, menu, new_mode):
+    def _overview_radio_clicked(self, checked, *, menu, new_mode):
```

Comment:
I would do
```suggestion
    def _overview_radio_clicked(self, checked=True, *, menu, new_mode):
```
just in case we need it for older versions of Qt or whatever

Reply:
I don't think we do, it's not a recent addition. 
Qt 5.9: https://doc.qt.io/archives/qt-5.9/qabstractbutton.html#clicked
Qt 4.8: https://doc.qt.io/archives/qt-4.8/qabstractbutton.html#clicked

Reply:
It makes me wonder how this ever worked...

Reply:
Never did? 😅 
That's also why I prefer to leave this without a default.. if for whatever reason it hides a bug further down the line.

Reply:
Might end up needing this default for the MNE testing suite (like above), we'll see I guess!

Reply:
Actually, the menu to select the overview mode was a drop-down list before, which explains why it used to work :)